### PR TITLE
#167733812 Create helper method for invalid user parameters

### PR DESCRIPTION
**Pull request by tamunoibi**
#### What does this PR do?
Setsup descriptive signup/Registration validation errors

#### Description of Task to be completed?
* Create a new branch for the feature
* Create file  /helpers/ValidationHelper.js
*  Import the required module - express validator
* Create helper method for invalid user parameters

#### How should this be manually tested?
A descriptive signup/Registration validation errors if you create a password that is less than 8 characters

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#167733812](https://www.pivotaltracker.com/story/show/167733812)

#### Questions:
N/A

### CODE REVIEW

**Comment by fegzycole** [comment]
Looks good

**Review by Ileriayo — CHANGES_REQUESTED** [review]
I noticed that  you pushed the `dist` folder. This must have been created when you ran `npm start` which is meant for production. Endeavor to use `npm run start-dev` for development. The `dist` folder is not supposed to be checked into version control. Include it in the `.gitignore` file and force push again.

**Comment by tamunoibi** [comment]
> I noticed that you pushed the `dist` folder. This must have been created when you ran `npm start` which is meant for production. Endeavor to use `npm run start-dev` for development. The `dist` folder is not supposed to be checked into version control. Include it in the `.gitignore` file and force push again.

**Comment by tamunoibi** [comment]
> > I noticed that you pushed the `dist` folder. This must have been created when you ran `npm start` which is meant for production. Endeavor to use `npm run start-dev` for development. The `dist` folder is not supposed to be checked into version control. Include it in the `.gitignore` file and force push again.

I have removed the dist folder. Thank you

**Review by Ileriayo — APPROVED** [review]
1. Great job setting up validations @girlking.

2. The class name `Helpers` & module name `Helper.js` is too generic. Try renaming it in your next PR.

File: .env.example [hunk 1]
```diff
@@ -0,0 +1 @@
+DATABASE_DEV=
```
Comment (Ileriayo) [inline .env.example:1]: Thanks for adding this file.

**Review by Ileriayo — APPROVED** [review]
Good job. However, you have to resolve pending merge conflicts @girlking.

**Comment by kingsley-einstein** [comment]
Looks ok

**Review by EziamakaNV — APPROVED** [review]
Looking good!

**Pull request merged**